app/views/camp/view.php

```diff
@@ -0,0 +1,154 @@
+
+            <ul class="nav navbar-nav navbar-left">
+                <li>
+                    <a href="#menu-toggle" id="menu-toggle">
+                        Modules
+                    </a>
+                </li>
+            </ul>
+
+        </div><!-- /.navbar-collapse -->
+    </div><!-- /.container-fluid -->
+</nav>
+<body style="padding: 50px">
+<div id="wrapper" class="toggled">
+    <!-- Sidebar -->
+    <div id="sidebar-wrapper">
+        <ul class="sidebar-nav">
+
+            <li>
+                <a href="<?php echo base_url(); ?>missing/">
+                    Missing People
+                </a>
+            </li>
+
+            <li>
+                <a href="<?php echo base_url(); ?>organisations/">
+                    Organisations
+                </a>
+            </li>
+
+            <li>
+                <a href="#">
+                    Camps
+                </a>
+            </li>
+
+            <li>
+                <a href="#">
+                    Assets
+                </a>
+            </li>
+
+            <li>
+                <a href="#">
+                    Requests
+                </a>
+            </li>
+
+        </ul>
+    </div>
+    <div id="page-content-wrapper" style="background-color: #ffffff;">
+        <div class="container-fluid">
+            <div class="row">
+                <div class="col-lg-12">
+                    <div class="page-header">
+                        <h1 style="text-align: center;">
+                           Camps List<br/>
+                        </h1>
+                        <br /><br />
+
+                        <?php
+                            foreach ($camp_list as $list) {
+                                echo "<div class='panel panel-primary'>
+                                      <div class='panel-heading' style='font-size: large; padding-bottom: 15px'>
+                                        <b>Camp Name: " . $list['name'] . "</b>
+                                    
+                                        <div class='btn btn-info show' style='float: right;'>Show More</div>
+                                      </div>
+                                      ";
+                                echo "<div class='panel-body' id='panelBody1' style='display: none;'>";
+                                echo "<ul class='list-group'>";
+                                echo "<li class='list-group-item'><b> Organisation Name: </b>" . $list['organisation_name'] . " </li><li class='list-group-item'><b>Camp_Head:  </b>".$list['fname']." ".$list['lname']."</li><li class='list-group-item'><b>Contact Number: </b>".$list['phone_no']."</li><li class='list-group-item'><b>Email:  </b>".$list['email']."</li>"."</li><li class='list-group-item'><b>Mailing List:  </b>".$list['mailing_list']."</li>"."</li><li class='list-group-item'><b>Mailing Address:  </b>".$list['mailing_address']."</li>"."</li><li class='list-group-item'><b>Gis URL:  </b>".$list['gis_url']."</li>";
```

Comment:
It is still not merged, so either we'll have to put this on hold or merge it change later.